lan867x/README.md

```diff
@@ -0,0 +1,44 @@
+# Microchip LAN867x Ethernet PHY Driver
+
+## Overview
+
+The LAN8670/1/2 is a high-performance 10BASE-T1S single-pair Ethernet PHY transceiver for 10 Mbit/s half-duplex networking over a single pair of conductors. 
+More information about the chip can found on the product page: [LAN8670](https://www.microchip.com/en-us/product/lan8670), [LAN8671](https://www.microchip.com/en-us/product/lan8671).
+
+## ESP-IDF Usage
+
+Just add include of `esp_eth_phy_lan867x.h` to your project,
```

Comment:
Should we also add something like "Add this component to your project using ```idf.py add-dependency``` command."? Please also update adin1200 README (do it in separate commit).

Reply:
For now I addressed it only in lan867x related README. Same for other changes which should be duplicated in ADIN1200.

Reply:
Why not to fix ADIN1200?

Reply:
Fixed both points in ADIN1200. Now a separate commit in the branch.